### PR TITLE
Added support for session and query report fragments.

### PR DESCRIPTION
**Pull request by ReigoHein**
Fixed some issues, separated the fragments into different fragment collection types.

### CODE REVIEW

File: viewer/public/common.js [hunk 1]
```diff
@@ -808,6 +808,42 @@ $(document).ready(function() {
     });
     return false;
   });
+  
+  //////////////////////////////////////////////////////////////////////////////////
+  // Reports
+  //////////////////////////////////////////////////////////////////////////////////
+  $("#queryReport").click(function (e) {
```
Comment (linphoebe) [inline viewer/public/common.js]: when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
$(".queryReport").click(function (e) {

File: viewer/public/common.js [hunk 2]
```diff
@@ -808,6 +808,42 @@ $(document).ready(function() {
...
+    }, function(qs, ids, filename) {
+      if (ids) {
+        qs.push({name: "ids", value: ids});
+      }
+      window.location = "queryReport/" + $(that).attr("data-filename") + "/" + filename + "?" + $.param(qs);
+    });
+    return false;
+  });
+  
+  $("#sessionReport").click(function (e) {
```
Comment (linphoebe) [inline viewer/public/common.js]: when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
$(".sessionReport").click(function (e) {

File: viewer/views/layout.jade [hunk 3]
```diff
@@ -159,6 +159,13 @@ html
         - if (molochClusters)
           each key in Object.keys(molochClusters)
             command.sendSessionButton(cluster=key,label="Send Session to #{molochClusters[key].name}")
+        - if (user.fragments && user.fragments.query)
+          - var keys = Object.keys(user.fragments.query).sort()
+          - if (titleLink == "sessionsLink" && keys.length > 0)
+            span 
+              b Reports
+            each key in keys
+              command#queryReport(label="#{user.fragments.query[key].name}", data-filename="#{user.fragments.query[key].identifier}")
```
Comment (linphoebe) [inline viewer/views/layout.jade]: when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
command(class="queryReport", label="#{user.fragments.query[key].name}", data-filename="#{user.fragments.query[key].identifier}")

File: viewer/views/layout.jade [hunk 4]
```diff
@@ -171,6 +178,13 @@ html
         - if (molochClusters)
           each key in Object.keys(molochClusters)
             command.sessionSendSessionButton.sendSessionAction(cluster=key,label="Send Session to #{molochClusters[key].name}")
+        - if (user.fragments && user.fragments.session)
+          - var keys = Object.keys(user.fragments.session).sort()
+          - if (keys.length > 0)
+            span 
+              b Reports
+            each key in keys
+              command#sessionReport(label="#{user.fragments.session[key].name}", data-filename="#{user.fragments.session[key].identifier}")
```
Comment (linphoebe) [inline viewer/views/layout.jade]: when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
command(class="sessionReport", label="#{user.fragments.session[key].name}", data-filename="#{user.fragments.session[key].identifier}")

**Comment by DesertStorm** [comment]
Thanks for catching that linphoebe, I've updated the branch and commited the fix.

**Pull request closed without merging**